cdap-ui/app/directives/timeline/timeline-ctrl.js

```diff
@@ -77,7 +77,7 @@ function TimelineController ($scope, LogViewerStore, LOGVIEWERSTORE_ACTIONS, myL
       names: ['system.app.log.error', 'system.app.log.warn', 'system.app.log.info', 'system.app.log.debug'],
       startTime : '',
       endTime : '',
-      resolution: '1m'
+      resolution: 'auto'
```

Comment:
Yes that's a fix recommended by the backend for this JIRA https://issues.cask.co/browse/CDAP-9248